tests/examples/schemas/booleans.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { test, testWithoutJS, expect } from 'tests/setup/tests'
+
+const route = '/examples/schemas/booleans'
+
+test('With JS enabled', async ({ example }) => {
+  const { button, page } = example
+  const mandatory = example.field('mandatory')
+  const optional = example.field('optional')
+  const nullable = example.field('nullable')
+  const defaultFalse = example.field('defaultFalse')
+  const defaultTrue = example.field('defaultTrue')
+
+  await page.goto(route)
+
+  // Render
+  await example.expectField(mandatory, { value: 'on', type: 'checkbox' })
+  await example.expectField(optional, {
+    value: 'on',
+    required: false,
+    type: 'checkbox',
+  })
+  await example.expectField(nullable, {
+    value: 'on',
+    required: false,
+    type: 'checkbox',
+  })
+  await example.expectField(defaultFalse, { value: 'on', type: 'checkbox' })
+  await example.expectField(defaultTrue, { value: 'on', type: 'checkbox' })
+
+  // Submit
+  await expect(button).toBeEnabled()
+  await button.click()
+
+  const actionResult = await page
+    .locator('#action-data > pre:visible')
+    .innerText()
+
+  await expect(actionResult).toContain('"mandatory": false')
+  await expect(actionResult).toContain('"nullable": null')
+  await expect(actionResult).toContain('"defaultFalse": false')
+  await expect(actionResult).toContain('"defaultTrue": true')
+
+  await page.reload()
```

Comment:
Either Playwright is finicky, or I didn't understand how it works.
We need a page reload here in order for it to understand the result has changed, even though the result did change when watching it on the debugger.
We also needed a second locator, because the first one always had the values of the first submit. Go figure.